kpconv_torch/datasets/S3DIS.py

```diff
@@ -842,6 +842,13 @@ def read_input(self, filepath, xyz_only=False):
             raise OSError(f"Unsupported input file extension ({file_extension}).")
         return points, colors, labels
 
+    def generate_projected_point_batches(self, file_idx, file_path, step=5_000_000):
```

Comment:
What does this syntax correspond to? `5_000_000` Is it an integer?

Reply:
That's a (more-readable) version of 5000000. You may find details about the underscores in numeric literals through the corresponding [PEP](https://peps.python.org/pep-0515/).